pkg/spdk/server.go

```diff
@@ -270,6 +334,7 @@ func (s *Server) verify() (err error) {
 				}
 			}
 			if noReplicaLvol {
+				logrus.Infof("[DEBUG]: delete replica %v from replicaMap %v", replicaName, replicaMap)
```

Comment:
```suggestion
				logrus.Debugf("[DEBUG]: delete replica %v from replicaMap %v", replicaName, replicaMap)
```

Reply:
This is just for debugging the unit test
will be removed when we are going to merge this PR.

Reply:
Removed.